docs/install-with-docker.md

```diff
@@ -315,22 +301,10 @@ This is an example of a fresh, empty AzerothCore server running with Docker on m
 
 When used on GNU/Linux system, the amount of memory used by Docker is even less.
 
-### Docker containers vs Virtual machines
-
-Using Docker will have the same benefits as using virtual machines, but with much less overhead:
-
-![Docker containers vs Virtual machines](https://user-images.githubusercontent.com/75517/51078179-d4fec680-16b1-11e9-8ce6-87b5053f55dd.png)
-
-_AzerothCore running on macOS with Docker_
-![AzerothCore on macOS using Docker](https://user-images.githubusercontent.com/75517/51341229-2089e980-1a91-11e9-8d06-ebd5897552d4.png)
-
-_AzerothCore running on Windows 10 with Docker_
-![AzerothCore on Windows 10 with Docker](https://user-images.githubusercontent.com/75517/51561998-966ec600-1e80-11e9-939e-d522c71de459.png)
```

Comment:
why did you remove this?

Reply:
Probably because it has nothing to do with building with docker

Reply:
The images aren't scaled well so the page layout can be hard to read.

I think this document should describe building with docker and addressing the few differences in running the server with docker, but I don't think that we should describe the benefits or standard procedures for docker